components/rewards/RewardsCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {
+  Card,
+  ContentContainer,
+  ContentText,
+  ContentText2,
+  StarIcon
+} from '../../styled/rewards';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Colors from '../../assets/Colors';
+
+class RewardCard extends React.Component {
+  render() {
+    return (
+      <Card>
+        <StarIcon>
+          <FontAwesome5 name="star" size={24} color={Colors.darkerGreen} />
```

Comment:
To make this solid, add the word solid here as a prop.
`<FontAwesome5 name="star" size={24} color={Colors.darkerGreen} />`

Also, the icon color is primaryGreen.